bluesky/plan_stubs.py

```diff
@@ -748,15 +768,19 @@ def inner_trigger_and_read():
             if hasattr(obj, 'trigger'):
                 no_wait = False
                 yield from trigger(obj, group=grp)
+        # read from the streaming devices
         yield from create(streaming_name)
         for obj in devices:
             reading = (yield from read_streaming(obj))
```

Comment:
No need to assign `yield from read_streaming(obj)` to a variable now that it isn't used.